WebKit export of https://bugs.webkit.org/show_bug.cgi?id=190642

woaiwyhty opened the pull request:
WebKit export from bug: [MediaRecorder should fire a stop event when all tracks are ended](https://bugs.webkit.org/show_bug.cgi?id=190642)

Ms2ger committed:
Add an instance of ImageData to the HTML interfaces test.

This causes the test to catch a bug in WebKit, where it defines a "data"
property on the instance, along with the correct property on the prototype.

Ms2ger committed:
Add some tests for invalid arguments to WebAssembly.*Streaming.

Ms2ger committed:
Test some invalid HTTP statuses.

Ms2ger committed:
Test cross-origin responses for WebAssembly.

Ms2ger committed:
Add tests for case differences in the wasm content type.

Ms2ger committed:
Add a test for empty bodies.

Ms2ger committed:
Add a test for disturbed bodies.

Ms2ger committed:
Add a test for passing a rejected Promise.

Ms2ger committed:
Add some tests for instantiateStreaming().

ericwilligers committed:
[cssom-view] Inheritance of CSSOM View properties

Properties inherit or not according to the spec.
Properties have initial values according to the spec.
https://drafts.csswg.org/cssom-view/#property-index

frivoal committed:
[css-text] Remove stale tests

This was written based on a previous version of the spec, and is no
longer correct.

css/css-text/overflow-wrap/overflow-wrap-break-word-002.html checks the
correct behavior instead.

Reported in https://github.com/w3c/csswg-drafts/issues/3221

emilio committed:
Honor will-change: position dynamic changes.

Differential Revision: https://phabricator.services.mozilla.com/D9032

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499863
gecko-commit: 2dd8df3604ef3a4130daea876a25efe6db16536f
gecko-integration-branch: autoland
gecko-reviewers: dbaron

Ms2ger commented on mediacapture-record/MediaRecorder-stop.html:
```diff
@@ -0,0 +1,37 @@
+        video.getVideoTracks()[0].stop();
+        t.step_timeout(() => {
+            assert_unreached("stop event is not fired after 2 seconds");
+        }, 2000);
```
Generally, we prefer just letting the test time out rather than adding an explicit timeout like this.

KWierso committed:
Make sure we don't try to remove device forwards that no longer exist

The stop() function is apparently being called multiple times at the end of a test run, causing us to sometimes attempt to remove device forwards that were removed in an earlier stop() attempt but before the conditions checked in stop() to remove the device forwards get changed. This caused a traceback to be printed out at the end of every test run, complaining about attempts to remove device forwards that don't exist.

Rather than dig through to figure out why/where stop() is getting called twice, this patch just papers over the issue by making sure there is at least one device forward still listed before attempting to remove them.

Differential Revision: https://phabricator.services.mozilla.com/D9175

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499889
gecko-commit: 6ff410edad90dbc51d72aeb722d0ff8d182606c1
gecko-integration-branch: autoland
gecko-reviewers: gbrown

dholbert committed:
Reduce text-sizing dependency & use clearer colors in reftest flex-flexitem-percentage-prescation.html

Commit:
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=190642

emilio committed:
Merge pull request #13630 from dholbert/master

Reduce text-sizing dependency & use clearer colors in reftest flex-flexitem-percentage-prescation.html

autofoolip committed:
Update interfaces/service-workers.idl (#13636)

Source: https://github.com/tidoust/reffy-reports/blob/d42d094/whatwg/idl/service-workers.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/443995922

foolip committed:
Add expectations for Safari in for infrastructure/ tests

In preparation for running these tests in CI.

Verified locally with Safari Technology Preview 67:
```
export no_proxy='*'
./wpt run --metadata infrastructure/metadata/ --channel=preview safari infrastructure/
./wpt run --metadata infrastructure/metadata/ --channel=preview safari_webdriver infrastructure/
```

With Safari 12 there are still failures, but we'll not use that in CI.

foolip committed:
Rename idlharness.window.js to idlharness.https.window.js (#13634)

In preparation for https://github.com/web-platform-tests/wpt/pull/13609.

autofoolip committed:
Update interfaces/webmidi.idl (#13609)

Source: https://github.com/tidoust/reffy-reports/blob/a37e00b/whatwg/idl/webmidi.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/443539880

frivoal committed:
Merge pull request #13589 from frivoal/text-3221

[css-text] Remove stale test

ehsan-karamad committed:
Reland "Implement fallback content for RemoteFrameOwner"

This is a reland of 02cb80e67052e59d324ae1f78d2c94c63da52939

The issues seems to have been the test case {'a', 'b', 'b'} where
apparently the failed navigation still fires a load event which might
arrive sooner than fallback update. The reland modifies the test file
to suppress sending 'OBJECT_LOAD' event from the test page.

Original change's description:
> Implement fallback content for RemoteFrameOwner
>
> RemoteFrameOwner does not implement the logic for fallback content. This
> means if a cross-origin navigation fails with some error, the owner
> element (in this context, <object>) in the parent process does not get
> notified and will not use its fallback content (instead the frame might
>  show an error page).
>
> When the <object> has fallback content, it should always use that over
> the frame's error message. To support this matter, this CL implements
> fallback methods in RemoteFrameOwner. Essentially,
>
>   * When <object> creates a local frame, the corresponding frame tree
> node will be marked as the type that "can" render fallback content. This
> will propagte everywhere using FrameReplicationState.
>
>   * When the provisional loading of a frame fails, RemoteFrameOwner will
> notify the browser through the current proxy for navigation. The browser
> then uses the parent frame to notify the renderer process that the owner
> element should render its own fallback content.
>
>   * If the <object> does not specify fallback, the navigation to error
> page commits and the error page is shown.
>
>   * When the <object> renders its own content, the remote frame stays
> alive. This is a bug but not a new one; It will be fixed in future CLs.
>
> Bug: 853140
> Cq-Include-Trybots: luci.chromium.try:linux_layout_tests_layout_ng
> Change-Id: Icad3934ccfc1823c0cdecd8e1223e6370ea4b3bb
> Reviewed-on: https://chromium-review.googlesource.com/c/1105635
> Commit-Queue: Ehsan Karamad <ekaramad@chromium.org>
> Reviewed-by: Daniel Cheng <dcheng@chromium.org>
> Reviewed-by: Ehsan Karamad <ekaramad@chromium.org>
> Reviewed-by: Wei Li <weili@chromium.org>
> Reviewed-by: Alex Moshchuk <alexmos@chromium.org>
> Cr-Commit-Position: refs/heads/master@{#601194}

TBR=dcheng@chromium.org,alexmos@chromium.org,weili@chromium.org

Bug: 853140
Change-Id: Ib1594aa642ea238f745fd29d191818fddc503afc
Cq-Include-Trybots: luci.chromium.try:linux_layout_tests_layout_ng
Reviewed-on: https://chromium-review.googlesource.com/c/1292899
Reviewed-by: Ehsan Karamad <ekaramad@chromium.org>
Commit-Queue: Ehsan Karamad <ekaramad@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601411}

ehsan-karamad committed:
Implement feature-policy: 'layout-animations'

This CL converts the current "animations" feature policy to
"layout-animations" feature policy: the former policy is about blocking
non-composited animations whereas the new policy only blocks a handful
of CSS animations which are known to (potentially) cause re-layout and
are also widely used. The current set of CSS properties considered in
the new policy is: {bottom, height, left, right,top, width}.

Bug: 874218
Change-Id: I97f08ce6b2b902c7057ea821416a9c145d26f068
Reviewed-on: https://chromium-review.googlesource.com/c/1235055
Commit-Queue: Ehsan Karamad <ekaramad@chromium.org>
Reviewed-by: Jeremy Roman <jbroman@chromium.org>
Reviewed-by: Robert Flack <flackr@chromium.org>
Reviewed-by: Ken Buchanan <kenrb@chromium.org>
Reviewed-by: Ian Clelland <iclelland@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601412}

cvazac committed:
fix them tests

foolip committed:
Simplify cases of `"%s" % already_a_string` in tools/ Python

rachelandrew committed:
[css-multicol] Adding assert statements to tests (#13616)

While inlining the tests into the multicol spec, I've been adding the missing assert statements. This is a set of those.

Ms2ger committed:
Update wasm-module-builder.js

From https://github.com/WebAssembly/spec/tree/64bf5c7a2447416f4690dae7c8ea9d7e2b841366/test/harness.

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1500442
gecko-commit: 081221c69c85d73b0ba0bde99b7128d62d2011fa
gecko-integration-branch: mozilla-inbound
gecko-reviewers: bbouvier

Ms2ger committed:
WebAssembly: Test aborted responses.

rayankans committed:
[Background Fetch] Fix match() crash.

Calling `match` was causing crashes in debug mode due to accessing
base::Optional<>::value without initialization. Calling match with
an unmatched request was crashing due to hitting a DCHECK. This returns
undefined now as per the spec.

Bug: 896768
Change-Id: I5d82e68ddca157a240ceaaec8990d2553366fbbb
Reviewed-on: https://chromium-review.googlesource.com/c/1289269
Commit-Queue: Rayan Kanso <rayankans@chromium.org>
Reviewed-by: Peter Beverloo <peter@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601541}

andreastt committed:
webdriver: write in-view centre point tests

These tests are not exhaustive, but covers some of the main use
cases and known edge cases.

Depends on D8880

Differential Revision: https://phabricator.services.mozilla.com/D8881

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499360
gecko-commit: d51f52d435daa3f87b398a7ca69879da8306b54a
gecko-integration-branch: autoland
gecko-reviewers: whimboo

andreastt committed:
webdriver: add interactability test for zero-sized element

Depends on D8881

Differential Revision: https://phabricator.services.mozilla.com/D9287

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499360
gecko-commit: 6afa28099f1ec6f3533128f3f8ac596c622a09a6
gecko-integration-branch: autoland
gecko-reviewers: whimboo

cbiesinger committed:
[css-flexbox] Apply min-height: auto to nested flexboxes again

This was made possible by crrev.com/c/1283482, which avoids the
performance and correctness issues this previously caused.

This relands crrev.com/c/1246730 / crrev.com/c/1252682. The ChromeOS UI
breakage was fixed by crrev.com/c/1281726 and crrev.com/c/1282462

IF THIS BREAKS ANY FURTHER CHROME UI:
Please don't revert this patch; instead, add min-height: 0 to any
inner nested flexboxes that may be affected by this patch. This is
an important change for web interop with the other browsers.

Bug: 596743
Change-Id: I9afe54dce82d41da452d1fdca8150ca22ebb6f9c
Reviewed-on: https://chromium-review.googlesource.com/c/1269235
Commit-Queue: Christian Biesinger <cbiesinger@chromium.org>
Reviewed-by: Morten Stenshorne <mstensho@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601540}

Ms2ger committed:
WebAssembly: Add tests for invalid code arguments.

inexorabletash committed:
Web Locks API: Fix IDL glitch

A callback defined in Web IDL should take a nullable argument. We
don't enforce this in our bindings code and it can't be introspected
from JS so this is not a testable or web-facing change, just aligning
the code with a spec fix.

Spec issue: https://github.com/WICG/web-locks/issues/51
Spec link: https://wicg.github.io/web-locks/#callbackdef-lockgrantedcallback

Change-Id: Icc2e076f62e45629a283d0039e2be30b4a6826c4
Reviewed-on: https://chromium-review.googlesource.com/c/1294421
Commit-Queue: Joshua Bell <jsbell@chromium.org>
Commit-Queue: Victor Costan <pwnall@chromium.org>
Reviewed-by: Victor Costan <pwnall@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601716}

Commit:
HttpStreamParser: Reject headers with nulls in them.

While the HTTP spec further limits what values are legal, nulls are
particularly concerning, and it's safest just to reject them. See
discussion here: https://github.com/whatwg/xhr/issues/165

Chrome will be the first browser to reject nulls in responses, despite
there being wpt tests for this, so we'll have to keep an eye out for
breakages.

For reference, 0x00 through 0x1F aren't allowed in header values or
fields, (https://tools.ietf.org/html/rfc7230#section-3.2 - VCHAR
excludes those characters).  CRs and LFs are of course needed, and
0x0C and 0x0B are allowed by other specs for particular
header parsers, strangely.

This CL does not affect other code that can generate HTTP response
headers, which still uses the old behavior of just removing nulls.
ServiceWorkers, extensions, WebPackages, Dial (?), and various tests
still inherit the old behavior, since they create headers directly
with a method that can't fail.  It does introduce a new helper method,
however, that they should eventually be switched to use:
HttpResponseHeaders::TryToCreate().  We should probably put off
conversion until this successfully makes it to stable.

Bug: 832086
Change-Id: Ib75ac03a6a298238cafb41eaa5f046c082fd0bdf
Reviewed-on: https://chromium-review.googlesource.com/c/1291812
Reviewed-by: Asanka Herath <asanka@chromium.org>
Commit-Queue: Matt Menke <mmenke@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601776}

foolip committed:
Move and fix the action_sequence documentation (#13648)

Commit:
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=190554

bzbarsky committed:
part 3.  Fix our DOM pre-insertion checks to actually match the spec.

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1500956
gecko-commit: bbb8bac97270807cf396a7d0c33a777e6a57d4d0
gecko-integration-branch: mozilla-inbound
gecko-reviewers: smaug

irori committed:
Move more Signed Exchange layout tests to WPT

Bug: 895723
Change-Id: If7918de7e3717b01619ee21fa87eeaffb971287c
Reviewed-on: https://chromium-review.googlesource.com/c/1292661
Reviewed-by: Kouhei Ueno <kouhei@chromium.org>
Reviewed-by: Tsuyoshi Horo <horo@chromium.org>
Commit-Queue: Kunihiko Sakamoto <ksakamoto@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601828}

TimothyGu committed:
Reland "bindings: Implement timers with V8Function"

This is a reland of 254369a5f6df06c2c6be067d14c2cb2a036ba173. It addresses bug
888025 by adding ASAN test expectations, as the relevant V8 feature does not
yet support running on ASAN builds.

Original change's description:
> bindings: Implement timers with V8Function
>
> This fixes bug 866610 by using the IDL infrastructure to properly enter
> the v8::Context before calling the registered callback.
>
> Also ensure eager finalization of ScheduledAction in DOMTimer to
> prevent a memory leak. Added two more effective DCHECKs to confirm.
>
> Bug: 866610
> Change-Id: I37d7bd05f035fe31856cfe68bae51aa0632cd3b1
> Reviewed-on: https://chromium-review.googlesource.com/1220486
> Reviewed-by: Nate Chapin <japhet@chromium.org>
> Reviewed-by: Yuki Shiino <yukishiino@chromium.org>
> Reviewed-by: Hitoshi Yoshida <peria@chromium.org>
> Commit-Queue: Timothy Gu <timothygu@chromium.org>
> Cr-Commit-Position: refs/heads/master@{#593108}

TBR=japhet@chromium.org

Bug: 866610, 888025
Change-Id: Iee5c1d6917ad7770383e06a425f96000835a663a
Reviewed-on: https://chromium-review.googlesource.com/c/1239624
Reviewed-by: Nate Chapin <japhet@chromium.org>
Reviewed-by: Hitoshi Yoshida <peria@chromium.org>
Reviewed-by: Kentaro Hara <haraken@chromium.org>
Reviewed-by: Yuki Shiino <yukishiino@chromium.org>
Commit-Queue: Timothy Gu <timothygu@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601830}

emilio committed:
Don't apply layer scaling to fallback filters.

Using layer scaling confuses the filter code (see bug 1497239).

Right now we're sort-of double-scaling. Looks like what the filter code expects
is just getting the scaling passed down, see bug 1224207.

Differential Revision: https://phabricator.services.mozilla.com/D8910

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1498221
gecko-commit: 3ef66c8d682c099b0ccb8bc41d2fd678dc39d29a
gecko-integration-branch: autoland
gecko-reviewers: mattwoodrow

dholbert committed:
Remove reference loop from test clip-path-content-syling.svg. (#13660)

TimothyGu committed:
Revert "Reland "bindings: Implement timers with V8Function""

This reverts commit 17f4e7d2ec13d6112dfcd8aa253f340bdb00d395.

Reason for revert: Still breaks ASAN

Original change's description:
> Reland "bindings: Implement timers with V8Function"
>
> This is a reland of 254369a5f6df06c2c6be067d14c2cb2a036ba173. It addresses bug
> 888025 by adding ASAN test expectations, as the relevant V8 feature does not
> yet support running on ASAN builds.
>
> Original change's description:
> > bindings: Implement timers with V8Function
> >
> > This fixes bug 866610 by using the IDL infrastructure to properly enter
> > the v8::Context before calling the registered callback.
> >
> > Also ensure eager finalization of ScheduledAction in DOMTimer to
> > prevent a memory leak. Added two more effective DCHECKs to confirm.
> >
> > Bug: 866610
> > Change-Id: I37d7bd05f035fe31856cfe68bae51aa0632cd3b1
> > Reviewed-on: https://chromium-review.googlesource.com/1220486
> > Reviewed-by: Nate Chapin <japhet@chromium.org>
> > Reviewed-by: Yuki Shiino <yukishiino@chromium.org>
> > Reviewed-by: Hitoshi Yoshida <peria@chromium.org>
> > Commit-Queue: Timothy Gu <timothygu@chromium.org>
> > Cr-Commit-Position: refs/heads/master@{#593108}
>
> TBR=japhet@chromium.org
>
> Bug: 866610, 888025
> Change-Id: Iee5c1d6917ad7770383e06a425f96000835a663a
> Reviewed-on: https://chromium-review.googlesource.com/c/1239624
> Reviewed-by: Nate Chapin <japhet@chromium.org>
> Reviewed-by: Hitoshi Yoshida <peria@chromium.org>
> Reviewed-by: Kentaro Hara <haraken@chromium.org>
> Reviewed-by: Yuki Shiino <yukishiino@chromium.org>
> Commit-Queue: Timothy Gu <timothygu@chromium.org>
> Cr-Commit-Position: refs/heads/master@{#601830}

TBR=peria@chromium.org,yukishiino@chromium.org,haraken@chromium.org,japhet@chromium.org,timothygu@chromium.org

Change-Id: Ie4f45dfcc1adcc2ac3469eab99dba813723288f4
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Bug: 866610, 888025
Reviewed-on: https://chromium-review.googlesource.com/c/1296057
Commit-Queue: Timothy Gu <timothygu@chromium.org>
Reviewed-by: Kentaro Hara <haraken@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601864}

ewilligers committed:
[css-multicol] Initial values and inheritance (#13651)

* [css-multicol] Initial values and inheritance

Test that CSS Multi-column Layout properties have the initial values
given in the spec. None of the properties inherit.

https://drafts.csswg.org/css-multicol/#property-index

Identified bugs:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19367855/
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19367975/
https://bugs.webkit.org/show_bug.cgi?id=190702

* Document the assert_inherited function arguments.

foolip committed:
Add tentative tests for selection.addRange() corner cases (#7596)

bakulf committed:
createImageBitmap should throw RangeError if sw or sh is 0

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1500734
gecko-commit: ce374302821c1a202263562910e7888ed455f5d8
gecko-integration-branch: mozilla-inbound
gecko-reviewers: aosmond

jgraham committed:
Update docker image to Ubuntu 18.04 (#13676)

This is the latest LTS release, so it makes sense to test on it. Also it is a requirement
to get servo builds running in the docker image since 16.04 has a too-old gstreamer.

CYBAI committed:
Assert play promise rejected caused by pause()

CYBAI committed:
Catch the rejected promise to avoid harness error

CYBAI committed:
Assert `decodeAudioData` not reach the `catch`

gsnedders committed:
Fix #13656: make percentage-heights-003 test definiteness (#13667)

pyup-bot committed:
Update mozdownload from 1.24 to 1.25 (#13659)

foolip committed:
Document review_requested@noreply.github.com (#13678)

foolip committed:
Fix the documentation of `action_sequence` somewhat (#13680)

foolip committed:
Remove unused `channel` argument in `Firefox.version` (#13642)

The only call site is in at the end of `setup_wptrunner` (run.py) where
no `channel` argument is passed. At that point, `setup_kwargs` has
already been called, which guarantees that `kwargs["binary"]` is set, so
even the fallback `self.find_binary(channel)` (where `channel` would be
`None`) is dead code.

mstensho committed:
Don't let outer multicol height confuse orphans/widows on inner multicol.

In the initial layout pass of a multicol container with auto height,
there'll be no known breaks, but we will honor the contraints of any
outer fragmentation context, and insert pagination struts if necessary.
These pagination struts need to be ignored when we calculate the height
requirements imposed by orphans and widows requirements, or we risk
making the inner multicol too tall.

Bug: 891703
Change-Id: Ia9281508fcbba3572d653f1bbf863f045db71827
Reviewed-on: https://chromium-review.googlesource.com/c/1296492
Commit-Queue: Morten Stenshorne <mstensho@chromium.org>
Commit-Queue: Emil A Eklund <eae@chromium.org>
Reviewed-by: Emil A Eklund <eae@chromium.org>
Cr-Commit-Position: refs/heads/master@{#601923}

mfalken committed:
service workers: Add long timeout to matchall test (#13684)

clients-matchall-order.https.html is a slow test because it does a lot of navigations.

andypaicu committed:
Added 304 CSP test and removed "content-*" prefix from headers to ignore on 304

Also updated kNonUpdatedHeaders with more headers from the
nsHttpResponseHead file

Spec: https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch
Spec issue: https://github.com/w3c/webappsec-csp/issues/161

While the spec does not give any list of content headers that should be ignored
on a 304 request, some of them are directly dependent on the resource body and
as such should not be updated (for example `content-length` cannot be different
since the content remains identical).

The exact list of ignored headers is identical to the one that firefox uses.

Bug: 174301
Change-Id: I8aab863b1f2733d051609e121539ad6acad36c6b
Reviewed-on: https://chromium-review.googlesource.com/c/1286427
Commit-Queue: Andy Paicu <andypaicu@chromium.org>
Reviewed-by: Mike West <mkwst@chromium.org>
Reviewed-by: Matt Menke <mmenke@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602001}

Commit:
Update isPointInFill/isPointInStroke to match spec adjustments

Rather than using the value of the 'pointer-events' property (and
indirectly, the 'visibility' property and potentially others not
explicitly listed in the spec), pass 'fill' and 'stroke' respectively to
better match spec [1].

Use AsPath() in isPointIn{Fill,Stroke} instead of going to the
LayoutSVGShape, to hopefully allow the LayoutSVGShape to be stricter
about invariants.

[1] https://svgwg.org/svg2-draft/types.html#InterfaceSVGGeometryElement

Bug: 896638
Cq-Include-Trybots: luci.chromium.try:linux_layout_tests_slimming_paint_v2;master.tryserver.blink:linux_trusty_blink_rel
Change-Id: I4f8ef5ae4e84d29187e68ca243e3344a9a27544c
Reviewed-on: https://chromium-review.googlesource.com/c/1288574
Commit-Queue: Fredrik Söderquist <fs@opera.com>
Reviewed-by: Philip Rogers <pdr@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602018}

jugglinmike committed:
[testharness.js] Correct typos

domfarolino committed:
Cross origin workers should fail to fetch (#13671)

* Cross origin workers should fail to fetch

* Pass tests when cross-origin workers fail to construct, and fix other tests

* Assert that DOMExceptions thrown on worker construction are SecurityErrors

* Add asserts to Worker/SharedWorker constructor tests, fix Worker tests

anforowicz committed:
CORB tests for JSON parser breakers.

Bug: 809261
Change-Id: I520ba33b5f314650370e7be7b50f809fc72de43d
Reviewed-on: https://chromium-review.googlesource.com/c/1295185
Commit-Queue: Łukasz Anforowicz <lukasza@chromium.org>
Reviewed-by: Charlie Reis <creis@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602183}

wangxianzhu committed:
[run_web_tests] Remove extra baselines after crrev.com/c/1277654

After crrev.com/c/1277654, we no longer generate text result of many
tests, thus these tests no longer need the text baselines.

This CL removes all the extra baselines, including the extra text
baselines after the above CL, and also other kind of extra baselines
(e.g. extra -expected.png for reftests).

Bug: 703899
Cq-Include-Trybots: luci.chromium.try:linux-blink-gen-property-trees;luci.chromium.try:linux_layout_tests_layout_ng;luci.chromium.try:linux_layout_tests_slimming_paint_v2;master.tryserver.blink:linux_trusty_blink_rel
Change-Id: Iadc647d0393e3f61a7d983bc3946042602de70c0
Reviewed-on: https://chromium-review.googlesource.com/c/1292599
Commit-Queue: Xianzhu Wang <wangxianzhu@chromium.org>
Reviewed-by: Ned Nguyen <nednguyen@google.com>
Cr-Commit-Position: refs/heads/master@{#602195}

kjd564 committed:
Updating tests for Window onerror is not triggered bug fix.

Chromium starts propagating errors from worker's onerror to window's onerror
after https://chromium-review.googlesource.com/c/chromium/src/+/1287208,
resulting in test failures for mixed-content/module-data-worker-import
tests that do not account for the thrown exception being sent to the window.

This CL adds preventDefault() to the worker.onerror() to handle the case of
uncaught exceptions.

Strictly speaking, this is not necessary according to the current spec, as
"worker's error event" fired at Step 12.1 of #run-a-worker
https://html.spec.whatwg.org/multipage/workers.html#run-a-worker
does not cause "report the error" and thus window's onerror.
But whether window.onerror is fired or not is not what the mixed-content tests
aim to test.

Change-Id: I571aaa85d90b5f933b6212691dcc783579b294be
Reviewed-on: https://chromium-review.googlesource.com/c/1294535
Commit-Queue: Katie Dillon <kdillon@chromium.org>
Reviewed-by: Hiroshige Hayashizaki <hiroshige@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602207}

bakulf committed:
CreateImageBitmap should handle OOM as InvalidState error

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1500780
gecko-commit: b273730d6149f687c46ed375ccc47674a3ef5afd
gecko-integration-branch: mozilla-inbound
gecko-reviewers: aosmond

bakulf committed:
CreateImageBitmap must ignore the Blob.type value

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1497925
gecko-commit: cdc621a21acc1cd486abd0342a5b3fec89f594a5
gecko-integration-branch: mozilla-inbound
gecko-reviewers: aosmond

KWierso committed:
Add 'sw-e10s' to run_info_extras

Differential Revision: https://phabricator.services.mozilla.com/D9585

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1501426
gecko-commit: dce16565d2a8c37e885b9e105ffcb78a0e6458db
gecko-integration-branch: autoland
gecko-reviewers: jmaher

birtles committed:
Compare absolute dot-product to 1.0 when interpolating quaternions

See the extended commit message for the following spec change:

  https://github.com/w3c/csswg-drafts/commit/6b36d41ebc2e790c86cc768a6ea1dcfa81fffe75

Basically, by failing to take the absolute value, for certain content we can end
up doing division by zero which will mean that the test included in this patch
will cause an assertion to fail in debug builds and return
"matrix(NaN, NaN....)" in release builds.

Differential Revision: https://phabricator.services.mozilla.com/D9618

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1500820
gecko-commit: 0ce248abad81e801972662b8c56a726f039a9cf0
gecko-integration-branch: autoland
gecko-reviewers: hiro

gsnedders committed:
Fix #13699: css-shapes parsing-utils.js doesn't restore fonts properly (#13703)

qiuzhong committed:
Add UCS-4 support check for Python in build-css-testsuites.sh (#13696)

CSS test suite build system requires Python UCS-4 support.

Add this check in css/build-css-testsuites.sh.

Fixes: #5504

javifernandez committed:
Add myself to suggested-reviewers in css/css-align (#13705)

beaufortfrancois committed:
[Picture-in-Picture] Test aspect ratio in web platform tests.

Change-Id: Ic0ab988927938f42234895d68e65779b9fbb4c5e
Reviewed-on: https://chromium-review.googlesource.com/c/1288430
Reviewed-by: Mounir Lamouri <mlamouri@chromium.org>
Reviewed-by: Jochen Eisinger <jochen@chromium.org>
Commit-Queue: François Beaufort <beaufort.francois@gmail.com>
Cr-Commit-Position: refs/heads/master@{#602325}

wangxianzhu committed:
[run_web_tests] Check for extra baselines

After https://chromium-review.googlesource.com/c/chromium/src/+/1277654,
we no longer generate text result of many tests, thus these tests no
longer need the text baselines.

This CL checks for extra baselines in the following cases:
1. Extra -expected.png (existing logic) for ref tests;
2. Extra -expected.txt, -expected.png, -expected.wav if the test doesn't
   generate the result;
3. Extra -expected.txt for all-pass testharness tests.

For 2, the extra baseline will be treated as a mismatch failure (which
can be interpreted as that the test doesn't generate the result, but
the baseline expects the result).

For 1 and 3, there are just warning messages because for now we don't
have a good way to present them in the layout test result page.

We have already had a similar logic in pre-submit for 3 and the new
check in run_web_tests is more accurate. For other types of extra
baselines, it's hard to implement the check in presubmit because it
needs more knowledge than static pre-submit knows.

Bug: 703899
Cq-Include-Trybots: luci.chromium.try:linux-blink-gen-property-trees;luci.chromium.try:linux_layout_tests_layout_ng;luci.chromium.try:linux_layout_tests_slimming_paint_v2;master.tryserver.blink:linux_trusty_blink_rel
Change-Id: I9016d5583791f179f1d51561245602af615d21de
Reviewed-on: https://chromium-review.googlesource.com/c/1286894
Commit-Queue: Xianzhu Wang <wangxianzhu@chromium.org>
Reviewed-by: Ned Nguyen <nednguyen@google.com>
Cr-Commit-Position: refs/heads/master@{#602378}

KWierso committed:
Fix spelling of "NotAllowedError"

Differential Revision: https://phabricator.services.mozilla.com/D9615

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499871
gecko-commit: 121ccc3d4437ee3a44aa29d4c94feb17073f815c
gecko-integration-branch: autoland
gecko-reviewers: achronop

KWierso committed:
Fix the test to use http://nonexistent.web-platform.test

Differential Revision: https://phabricator.services.mozilla.com/D9263

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1499900
gecko-commit: dae5b4b90a4209aea03c26937ffb3a71ce0767c8
gecko-integration-branch: autoland
gecko-reviewers: jgraham

natechapin committed:
Expect CanvasGradient and CanvasPattern to be exposed in workers in workers/semantics/interface-objects

The spec has had them exposed on workers since
https://github.com/whatwg/html/commit/aedc72daef50aa472b36f158633eb339e7d5ca20

Bug: 655458
Change-Id: Ibb9c161d430cbdfeeb3afde1922bf96494c218f6
Reviewed-on: https://chromium-review.googlesource.com/c/1297284
Commit-Queue: Hiroki Nakagawa <nhiroki@chromium.org>
Reviewed-by: Hiroki Nakagawa <nhiroki@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602576}

Commit:
Remove alexshalamov from the suggested reviewers lists (#13706)

Alex is no longer working on those things, so drop him from those lists to
avoid confusion.

youennf committed:
Merge pull request #13669 from alanwaketan/wpt-export-for-webkit-190554

WebKit export of https://bugs.webkit.org/show_bug.cgi?id=190554

ewilligers committed:
[css-text] Inheritance and initial values (#13110)

Test that CSS Text properties inherit.
Test that they have the initial values given in the spec.
https://drafts.csswg.org/css-text/#propdef-white-space

ewilligers committed:
[css-ui] Inheritance and initial value (#13095)

Test that CSS Basic User Interface properties inherit.
Test that initial values match the spec.
https://drafts.csswg.org/css-ui/#property-index

bakulf committed:
Fix a wrong use of assert_throws in createImageBitmap-invalid-args

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1501924
gecko-commit: 5e6bc808c1d7bae1b7c63a257399820c67081c2c
gecko-integration-branch: mozilla-inbound
gecko-reviewers: Ms2ger

d0iasm committed:
ServiceWorker: Add new tests for updating the registration with
different script type.

These tests check that a registration is updated correctly with
different script type. At first Service Worker is register as classic
script type, then it is re-registered as module script type, and
vice versa.　A main　script is also updated at the same time.

Bug: 824647
Change-Id: I3ab9391791508662de8dc73ac932a6105c4d7fe6
Reviewed-on: https://chromium-review.googlesource.com/c/1295763
Commit-Queue: Asami Doi <asamidoi@google.com>
Reviewed-by: Matt Falkenhagen <falken@chromium.org>
Reviewed-by: Hiroki Nakagawa <nhiroki@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602650}

autofoolip committed:
Update interfaces/cssom-view.idl (#13716)

Source: https://github.com/tidoust/reffy-reports/blob/a9f000c/whatwg/idl/cssom-view.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/446017586

Commit:
Insert an error message block when parsing was stopped after an error

The <parsererror> element was not being added to the parsed document in
certain cases involving attribute parsing. For example, if an undeclared
namespace prefix was used in the document, then parsing would stop, but
a <parsererror> element would not be inserted into the document to
indicate that a parsing error occurred.

When XMLDocumentParser::StopParsing() is called to stop parsing, call
InsertErrorMessageBlock() to insert the <parsererror> element if an
error occurred.

Based on https://codereview.chromium.org/771493002/ by dtrebbien@gmail.com

Bug: 295809, 898775
Change-Id: Ic67a3ab885fae4b01ca92f6d927b3f523f7423ed
Reviewed-on: https://chromium-review.googlesource.com/c/1224791
Commit-Queue: Fredrik Söderquist <fs@opera.com>
Reviewed-by: Daniel Cheng <dcheng@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602674}

uysalere committed:
Add more tests covering constraints usage in getDisplayMedia()

Bug: 326740
Change-Id: I923fb55c07424573d2a2829f8c5ef22fc4e28466
Reviewed-on: https://chromium-review.googlesource.com/c/1292371
Reviewed-by: Guido Urdaneta <guidou@chromium.org>
Reviewed-by: Henrik Boström <hbos@chromium.org>
Commit-Queue: Emircan Uysaler <emircan@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602709}

docfaraday committed:
Fix bugs where transceivers could be associated with m-sections of another type.

Bug 1496245 - Part 0: web-platform-test for rollback followed by SRD with different media type.

Bug 1496245 - Part 1: Allow SRD(offer) to stomp un-negotiated level mappings on our transceivers.

Bug 1496245 - Part 2: Don't create extra datachannel transceivers, just reuse (and possibly re-enable) the one we have.

Differential Revision: https://phabricator.services.mozilla.com/D8259

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1496245
gecko-commit: d7641a700ff64f760f0f8483a458f9870418141f
gecko-integration-branch: autoland
gecko-reviewers: mjf, jib

zakerinasab committed:
Copy srgb/e-srgb color managed canvas layout tests to WPT

Bug: 898846
Change-Id: I16d7c18f111040c401e19eb2c77f1e6fec1c6200
Reviewed-on: https://chromium-review.googlesource.com/c/1299535
Reviewed-by: Fernando Serboncini <fserb@chromium.org>
Commit-Queue: Mohammad Reza Zakerinasab <zakerinasab@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602801}

sunyunjia committed:
Update scroll-padding initial values in wpt tests.

There was a conflict in the scroll snap spec for the initial values of
scroll-padding-top/bottom/left/right[1]. Some of the initial values were
auto, while others were 0px. Now the conflict has been resolved with
all the initial values set to auto[2], we should update the wpt test to
reflect that.

The current implementation in Blink already reflects the updated spec,
so this patch also removes the expectations that were simply for the
failures.
[1] https://github.com/w3c/csswg-drafts/issues/3189
[2]
https://drafts.csswg.org/css-scroll-snap-1/#propdef-scroll-padding-top

Bug: 891954
Change-Id: Ib9c2280590091be27127abeee0a611b687cd37a2
Reviewed-on: https://chromium-review.googlesource.com/c/1299605
Reviewed-by: Eric Willigers <ericwilligers@chromium.org>
Commit-Queue: Sandra Sun <sunyunjia@chromium.org>
Cr-Commit-Position: refs/heads/master@{#602929}

perryjiang committed:
move test_update_missing_imported_script.html to WPT

Differential Revision: https://phabricator.services.mozilla.com/D6439

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1357290
gecko-commit: 9f3a74845bdb0e7acb639c4ada4dc9c61280532d
gecko-integration-branch: autoland
gecko-reviewers: edenchuang

pyup-bot committed:
Update mozprofile from 2.0.0 to 2.1.0 (#13700)

pyup-bot committed:
Update mozdebug from 0.1 to 0.1.1 (#13707)

pyup-bot committed:
Update mozrunner from 7.1.0 to 7.2.0 (#13708)

foolip committed:
Avoid the use of `readlink -f` in shell scripts (#13639)

This doesn't work on macOS. Replacement taken from:
https://serverfault.com/a/406371

sideshowbarker committed:
cc Update makefile for doc-conformance tests

kojiishi committed:
[LayoutNG] Rebaseline fast/writing-mode/border-image-vertical-lr.html

fast/writing-mode/border-image-vertical-lr.html produces
different results because, in legacy, block direction borders
applied to inline boxes affect positioning of the line box in
vertical-lr writing-mode, when they should not.

This patch rebaselines it, and also adds a minimized test to
check such borders do not affect line box positioning.

Gecko, and WebKit fail the test. Edge passes this test,
though it paints the border at the incorrect position.
Testing the border positoin is not ref-testable and that it
is not included.

Bug: 636993
Change-Id: I2568d19dac7f7ea886a46d45620bc22feb172c53
Reviewed-on: https://chromium-review.googlesource.com/c/1301698
Reviewed-by: Emil A Eklund <eae@chromium.org>
Commit-Queue: Emil A Eklund <eae@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603051}

beaufortfrancois committed:
Fix pip aspect ratio test

dtapuska committed:
Remove tentative from test names for PostMessage options.

Now that the change has been merged to the HTML spec we can remove
the suffix.

Fixes web-platform-tests/wpt/issues/13736

BUG=846858

Change-Id: Ic12b90c44126f801827bba7057a53fe640fa2859
Reviewed-on: https://chromium-review.googlesource.com/c/1301841
Reviewed-by: Mustaq Ahmed <mustaq@chromium.org>
Commit-Queue: Dave Tapuska <dtapuska@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603108}

longsonr committed:
PutForwards is misspelled as PutsForward in SVG.idl

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1502250
gecko-commit: 1d7213c00697fd4589fd96401c527f8c5d420eaf
gecko-integration-branch: mozilla-inbound
gecko-reviewers: dholbert

Zirro committed:
Add missing window properties

ewilligers committed:
[css-transforms] Transforms initial values (#13023)

Test the initial values and non-inheritance of
CSS Transforms and CSS Motion Path properties.

https://drafts.csswg.org/css-transforms/#property-index
https://drafts.csswg.org/css-transforms-2/#property-index
https://drafts.fxtf.org/motion-1/#property-index

Browsers currently report 'flat' as the initial value of
transform-style, contrary to spec.

ewilligers committed:
[svg] circle radius 0 disables rendering (#13749)

A value of 0 for circle r, ellipse rx/ry, rect width/height
disables rendering.

https://www.w3.org/TR/SVG2/shapes.html#CircleElement
https://www.w3.org/TR/SVG2/shapes.html#EllipseElement
https://www.w3.org/TR/SVG2/shapes.html#RectElement

progers committed:
Update comments and test mix-blend-mode on pattern

The working group recently resolved that mix-blend-mode should not
apply to never-rendered elements such as pattern, see:
https://github.com/w3c/fxtf-drafts/issues/309. This patch updates the
comment in LayoutObject::IsBlendingAllowed and adds a web platform test
(all browsers should pass this already). This removes the TODO added in
https://chromium-review.googlesource.com/c/chromium/src/+/1169624.

Change-Id: Ie74fd48075ae377636cb3efe93dd2fa6896fc534
Reviewed-on: https://chromium-review.googlesource.com/c/1303954
Reviewed-by: Fredrik Söderquist <fs@opera.com>
Reviewed-by: Xianzhu Wang <wangxianzhu@chromium.org>
Commit-Queue: Philip Rogers <pdr@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603379}

Commit:
Force line-height:normal for themed comboboxes for compat with other UAs.

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1501908
gecko-commit: 89697dbd654a76380a910b5dc4c6c0f3afbaddc6
gecko-integration-branch: mozilla-inbound
gecko-reviewers: emilio

emilio committed:
Remove SVG use attribute change handling code from nsSVGUseFrame to SVGUseElement.

Most of those shouldn't rely on a frame to run.

Differential Revision: https://phabricator.services.mozilla.com/D9996

bugzilla-url: https://bugzilla.mozilla.org/show_bug.cgi?id=1502658
gecko-commit: 627720590ce22dcf92d9ffc0ac3a7ec3d1a45f84
gecko-integration-branch: autoland
gecko-reviewers: longsonr

ewilligers committed:
[filter-effects] filter arguments optional (#13750)

grayscale() etc may be used without arguments.

https://github.com/w3c/fxtf-drafts/issues/1

ewilligers committed:
[filter-effects] computed filter (#13751)

Omitted values are included and compute to their defaults.
https://drafts.fxtf.org/filter-effects-1/#computed-values-of-filter-functions
https://github.com/w3c/fxtf-drafts/issues/1

marcoscaceres committed:
Swapping @MSFTkihans with @zouhir (#13733)

We agreed to this at TPAC

beaufortfrancois committed:
Merge pull request #13734 from web-platform-tests/beaufortfrancois-patch-1

Fix pip aspect ratio test

zcorpan committed:
[docs] Embed a video with intro to wpt

lukebjerring committed:
Ignore includers of untested mixins (#13611)

skyostil committed:
WPT: Adjust expected timing resolution

We recently increased[1] the minimum resolution of web-exposed timers.
This patch relaxes the WPT tests accordingly to allow for a higher
resolution (with a small delta to account for floating point errors).

Bug: 897775

[1] https://chromium-review.googlesource.com/c/1290916

Change-Id: I6f8a8d8ac2b827ae54718db6f8940cafb1cad665
Reviewed-on: https://chromium-review.googlesource.com/c/1298017
Reviewed-by: Nicolás Peña Moreno <npm@chromium.org>
Commit-Queue: Sami Kyöstilä <skyostil@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603461}

mstensho committed:
Use the correct out-of-flow containing block in continuations.

We cannot just use the innermost inline, without checking if it's a
valid containing block for absolutely positioned descendants. Find the
right ancestor.

The idea of using the *next* inline in the continuation chain may not be
ideal, but it's not obvious that using the previous one would be much
better (it's more work to find that one anyway), so leaving it as it is.
The spec says that if the relatively positioned inline occurs on
multiple lines, CSS 2 doesn't define exact behavior. Therefore, in the
test, put the same amount of stuff on the line preceding the
interrupting block-level element as on the line following it, to be sure
that the abspos descendant is sized the way we want it to be.

Bug: 897535
Change-Id: Ie9bc4e83400bc283b9c95aafef821000f2303d12
Reviewed-on: https://chromium-review.googlesource.com/c/1301457
Commit-Queue: Morten Stenshorne <mstensho@chromium.org>
Reviewed-by: Emil A Eklund <eae@chromium.org>
Reviewed-by: vmpstr <vmpstr@chromium.org>
Reviewed-by: Mason Freed <masonfreed@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603466}

ericwilligers committed:
CSS: Move :where test to WPT

https://drafts.csswg.org/selectors-4/#zero-matches

BUG=898215

Change-Id: I262b0ea290b340b634c6c4ffe98fa425bcff2b84
Reviewed-on: https://chromium-review.googlesource.com/c/1303356
Reviewed-by: Rune Lillesveen <futhark@chromium.org>
Commit-Queue: Eric Willigers <ericwilligers@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603506}

Ms2ger committed:
Python 3: Use six.create_bound_method() in test_response.py (#13624)

notriddle committed:
Assign a name to iframes when loading the initial about:blank

Before, it would assign the name too late,
causing scripts (which will not wait for another tick)
to accidentally spawn pop-up windows instead of loading
into the iframe.

notriddle committed:
Use `t.step_func` and expected path checking

andruud committed:
[css-properties-values-api] Proper computed value for <color>.

As of an imminent spec change, <color> values no longer compute to their
specified value, but behave like <color> values in general.

Note that color keywords are parsed as CSSIdentifierValue (holding a
CSSValueID), and custom idents are parsed as CSSCustomIdentValue (holding a
String), which is why the "tomato | <color>" works as it should.

R=futhark@chromium.org

Bug: 641877
Change-Id: I946536a9d54dbaa7af589cb99acdba72f37fe016
Reviewed-on: https://chromium-review.googlesource.com/c/1303365
Reviewed-by: Rune Lillesveen <futhark@chromium.org>
Commit-Queue: Anders Ruud <andruud@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603628}

andruud committed:
[css-properties-values-api] Compute <angle> values to 'deg'.

R=futhark@chromium.org

Bug: 641877
Change-Id: I52878c9bca16765d3b32127b5637a7ec521e76dc
Reviewed-on: https://chromium-review.googlesource.com/c/1304416
Commit-Queue: Anders Ruud <andruud@chromium.org>
Reviewed-by: Rune Lillesveen <futhark@chromium.org>
Cr-Commit-Position: refs/heads/master@{#603672}

ewilligers committed:
[filter-effects] Inheritance and initial values (#13105)

Test that Filter Effects properties inherit.
Test that initial values match the spec.
https://drafts.fxtf.org/filter-effects/#property-index

Commit:
Merge branch 'wpt-export-for-webkit-190642' of github.com:woaiwyhty/wpt into local190642

Commit:
localchange1

foolip commented:
@woaiwyhty @Ms2ger do you know what happened here? The test changes were landed in WebKit in https://trac.webkit.org/changeset/237311/webkit but this was never merged...

foolip commented:
Sorry for the spam everyone, https://github.com/web-platform-tests/wpt-pr-bot/issues/38 is the issue about why @wpt-pr-bot does stuff with closed PRs when it shouldn't.

foolip commented:
Looks like mediacapture-record/MediaRecorder-stop.html (the only file originally in this PR) was later exported in https://github.com/web-platform-tests/wpt/pull/13805. So this should remain closed. Pardon again everyone for the spam. I'll remove the labels once again and then make sure to do nothing at all to avoid triggering @wpt-pr-bot again. This PR should remain closed and no additional action is needed.

Ms2ger commented:
@foolip I guess it was forgotten; stuff is still super-manual in WebKit, as you know. What did you do, though???

foolip commented:
Now that https://github.com/web-platform-tests/wpt-pr-bot/issues/38 I can (hopefully) explain what happened here without the bot adding back all the reviewers and comments.

On this PR, a rebase or merge went wrong and lots of files were considered affected. wpt-pr-bot labeled and added reviewers accordingly. The issue of >100 labels was addressed in https://github.com/web-platform-tests/wpt-pr-bot/issues/35, but things could still go wrong like this again.

Anyway, no action is needed here, because the changes were landed in another PR already.
